deploy/charts/cert-manager/values.yaml

```
@@ -928,66 +928,66 @@ cainjector:
     readOnlyRootFilesystem: true
 
   podDisruptionBudget:
-    # Enable or disable the PodDisruptionBudget resource
+    # Enable or disable the PodDisruptionBudget resource.
     #
     # This prevents downtime during voluntary disruptions such as during a Node upgrade.
     # For example, the PodDisruptionBudget will block `kubectl drain`
     # if it is used on the Node where the only remaining cert-manager
     # Pod is currently running.
     enabled: false
 
-    # Configures the minimum available pods for disruptions. Can either be set to
+    # It configures the minimum available pods for disruptions. It can either be set to
     # an integer (e.g. 1) or a percentage value (e.g. 25%).
     # Cannot be used if `maxUnavailable` is set.
     # +docs:property
     # minAvailable: 1
 
-    # Configures the maximum unavailable pods for disruptions. Can either be set to
+    # it configures the maximum unavailable pods for disruptions. It can either be set to
```

suggestion: As above, but if we don't change the start we at least need to capitalize `it`.

```suggestion
    # `maxUnavailable` configures the maximum unavailable pods for disruptions. It can either be set to
```